content/_index.nl.md

```diff
@@ -1,42 +1,48 @@
-intro: "Wij maken de digitale wereld veiliger door te scannen naar
-  kwetsbaarheden in digitale systemen en deze vervolgens te melden aan de mensen
-  die deze kunnen oplossen. "
+opener: Een waakzame speler in de digitale wereld
+intro: Het is ons doel om de digitale veiligheid te verbeteren door
+  kwetsbaarheden die we ontdekken in digitale systemen te melden aan partijen
+  die ze kunnen aanpakken en oplossen.
 fold:
-  showcase: /newsroom/articles/operation-endgame-divd-2024-00019
+  showcase: /newsroom/articles/_index
   received: Heb je een email van ons csirt team ontvangen? Lees hier verder
   received_alt: Afbeelding van een enveloppe
-  family: Wordt onderdeel van de DIVD en help ons de digitale wereld veiliger te maken
+  family: Wordt onderdeel van DIVD en help ons de digitale wereld veiliger te maken
   family_alt: Plaatje van een groep mensen die de DIVD familie moeten voorstellen
 whatwedo:
   t1_alt: Plaatje van een vergrootglas boven een waarschuwingssymbool
   t2_alt: Tekening van een persoon met een laptop voor een scherm met code erop.
   t3_alt: Plaatje van een vergrootglas boven een waarschuwingssymbool
-  title: Wat doen we?
+  Alt text for t1 image: Illustratie van een vergrootglas met een schild in het
+    midden dat een uitroepteken bevat
+  Alt text for t2 image: Illustratie van een wolk met uitroepteken
+  Alt text for t3 image: "Illustratie van een persoon met een laptop "
+  title: Wat we doen
   t1_image: images/global/scout.png
   t2_image: images/global/assessing.png
   t3_image: images/global/scout.png
-  description: "Wij scannen ip-adressen op het internet naar bekende, en
-    onbekende, kwetsbaarheden op het internet. Dit doen wij samen met ons team
-    van security researchers. Vrijwillig. Omdat wij geloven in een veiliger
-    internet. "
-  t1: Tag 1
-  t2: Tag 2
-  t3: Tag 3
+  description: "DIVD wil de digitale wereld veiliger maken door kwetsbaarheden die
+    we vinden in digitale systemen te rapporteren aan de mensen die ze kunnen
+    verhelpen. Wij doen dit door het scannen van ip-adressen naar bekende, en
+    onbekende kwetsbaarheden. DIVD heeft een wereldwijd bereik, maar we doen het
+    op z'n Hollands: open, eerlijk, collaboratief en gratis. "
+  t1: Het internet scannen op kwetsbaarheden
+  t2: De kwetsbaarheid melden aan de juiste mensen
+  t3: 0-Day disclosure
 helpushelpyou:
-  title: Security.txt, kleine moeite groot effect
-  description: Make our work easier by putting our security.txt in the code of
-    your website and our IP 194.5.73.0-255 on your allow list. This way you make
-    sure that our research team can look for vulnerabilities without alarming
-    your cyber security systems. And we know who to talk to when we find
-    vulnerabilities.
+  title: Security.txt. Kleine moeite, groot effect!
+  description: Maak ons werk gemakkelijker door onze security.txt in de code van
+    je website te zetten en ons IP 194.5.73.055 op je allow list. Dit zorgt
```

Comment:
Is dit bewust `194.5.73.055` aangezien we in het Engels een range noemen? `194.5.73.0-255`

Reply:
Geen idee, dat stukje heb ik letterlijk overgenomen.